.github/workflows/scripts/add-post-merge-comment.php

```diff
@@ -57,8 +57,8 @@
 
 $comment_body = "Hi @$merger_user_name, thanks for merging this pull request. Please take a look at these follow-up tasks you may need to perform:
 
-- [ ] Add the `status: needs changelog` label
-- [ ] Add the `status: needs testing instructions` label";
+- [ ] Add the `release: add changelog` label
+- [ ] Add the `release: needs testing instructions` label";
```

Comment:
This one has been changed to `release: add testing instructions`

Reply:
So it has ... [updated](https://github.com/woocommerce/woocommerce/pull/31908/files#diff-78d5fda19e0d8f115b992ba933d5855a258b0c824b5187f9dd72e50bac190f40R61).